src/main/python/apache/aurora/config/thrift.py

```
@@ -355,3 +359,19 @@ def not_empty_or(item, default):
       cronCollisionPolicy=select_cron_policy(job.cron_collision_policy()),
       taskConfig=task,
       instanceCount=numberOfInstances)
+
+def convert_health_check(job_health_check):
```

Might want to raise an Invalid Config if both health_ckecks are set